### ISSUE TITLE
Portrait screen is blank if I load an old saved game

### ISSUE THREAD

**Issue by JamesKrolak** [BarleyFlour/VisualAdjustments2#68]
**Describe the bug**
The Portrait screen is completely blank when I load some older saved games.  It works just fine on new games and saved games that are in the introduction.  In fact, I can load a save for this game from somewhere way back in Act 1 and the Portraits screen loads just fine.  

**To Reproduce**
1. Load up old saved game
2. Open Character panel
3. Click on Visual tab
4. Click on Portrait tab

**Expected behavior**
Portraits and options to select them should appear

**Screenshots**
![portrait screen](https://user-images.githubusercontent.com/38671509/229379811-f609256b-ee0e-4def-8232-7569bf680fbf.jpg)

**Game Info:**
 - Version: 2.1.1i or 2.0.7o

**Additional context**
Found in player.log right after opening the Portraits screen:

[VisualAdjustments2] [Error] System.NullReferenceException: Object reference not set to an instance of an object
  at VisualAdjustments2.UI.VoicePickerVM.SetupSelected () [0x00084] in :0 
  at VisualAdjustments2.UI.PortraitPickerVM.OnCharacterChanged (Kingmaker.EntitySystem.Entities.UnitReference data) [0x0004f] in :0 
  at UniRx.Observer+Subscribe`1[T].OnNextInternal (T value) [0x00008] in :0 
  at UniRx.Observer+BaseObserver`1[T].OnNext (T value) [0x00000] in :0 
  at UniRx.ReactiveProperty`1[T].Subscribe (System.IObserver`1[T] observer) [0x00014] in :0 
  at UniRx.ObservableExtensions.Subscribe[T] (System.IObservable`1[T] source, System.Action`1[T] onNext) [0x00011] in :0 
  at VisualAdjustments2.UI.PortraitPickerVM..ctor () [0x0013d] in :0 
  at VisualAdjustments2.UI.ServiceWindowsVMModified.ShowWindow (VisualAdjustments2.VisualWindowType type) [0x0010f] in :0

**Comment by Valdacil** [comment]
I am also having this trouble.  Loaded up a saved game in the middle of Act 3 and NONE of the tabs work.  Can't adjust doll, EE, anything.

**Comment by JamesKrolak** [comment]
> I am also having this trouble. Loaded up a saved game in the middle of Act 3 and NONE of the tabs work. Can't adjust doll, EE, anything.
> 
> I just used the mod successfully on a playthrough for Treasures DLC yesterday however. So mod itself is working under that mode.

Are you able to interact with the tabs other than Portrait and they just don't end up making changes to your character?  Or do they not respond at all?  I'm asking 'cuz I thought shortly after making my bug post that the entire mod wasn't working 'cuz anything I did on the EE tab, Doll tab, etc. wouldn't apply to my character.  But I figured out that my Mythic path's visual changes were overriding everything.  I'm a Lich, so my character's skin was dark gray and her hair was white.  I had to find another mod to disable all the Mythic path visual changes and then I could get Visual Adjustments 2 to change my character.  (Portrait screen is still blank, though, so I have to use Arcemi's Save Game Editor to change that)

**Comment by Valdacil** [comment]
No, for my Angel playthrough (midway through Act 3), all of the VA2 pages are blank.  I can switch to Visual and see the tabs there, but they are empty... just a blank beige space.  I can click the various tabs and the interface makes a noise like it is switching, but all tabs remain blank.
